src/python/grpcio/grpc/_channel.py

```diff
@@ -1439,15 +1588,19 @@ def _unsubscribe(state, callback):
                 break
 
 
-def _augment_options(base_options, compression):
+def _augment_options(
+        base_options: Optional[ChannelArgumentType],
+        compression: Optional[grpc.Compression]) -> Sequence[Tuple[str, Any]]:
```

Comment:
`Sequence[Tuple[str, Any]]` is (currently) the same as `ChannelArgumentType`.

Reply:
You're right, fixed :)